webapp/src/app/models/CourseImp.ts

```
@@ -132,4 +132,12 @@ export class CourseImp implements Course {
   getNumberUsers(): number {
     return this.numberUsers;
   }
+
+  setUrl(url: string) {
+    url ? (this.url = url) : (this.url = null);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)